src/Faker/Provider/hi_IN/Person.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Faker\Provider\hi_IN;
+
+class Person extends \Faker\Provider\Person
+{
+    
+    
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{middleNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{middleNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    /**
+     * @link http://www.indiaonlinepages.com/babynames/
```

Comment:
the content of this site is copyrighter. Please find a source compatible with MIT.